bower.json

```diff
@@ -5,7 +5,7 @@
     "video-js": "5.13.1",
     "videojs-youtube": "raccoongang/videojs-youtube#playback-rate-fix",
     "videojs-wistia": "raccoongang/videojs-wistia#playback-toggle-fix",
-    "videojs-transcript": "0.8.0",
+    "videojs-transcript": "xahgmah/videojs-transcript#master",
```

Comment:
We already have https://github.com/raccoongang/videojs-transcript
Please merge your fix into that repo.